examples/crm/src/login/LoginForm.tsx

```diff
@@ -7,7 +7,7 @@ import {
     CircularProgress,
     Typography,
 } from '@mui/material';
-import { Form, useLogin, useNotify, useSafeSetState } from 'ra-core';
+import { Form, useLogin, useNotify, useState } from 'ra-core';
```

Comment:
That's wrong, `ra-core` doesn't export a `useState` hook.

Reply:
fixed!